README.md

```diff
@@ -96,3 +96,17 @@ See [our other projects][community] or [hire our team][hire] to help build your
 
 [community]: https://github.com/nimblehq
 [hire]: https://nimblehq.co
+
+## FAQ
+
+### 1. I'm having Mix, Elixir, and Erlang/OTP but can not initialize project by the raise of the error `(Mix) The task "phx.new" could not be found`
```

Comment:
How about just saying 

### 1. Getting `(Mix) The task "phx.new" could not be found` error.

The Phoenix application generator is missing, install it by

```bash
# Latest Phoenix version
mix archive.install hex phx_new

# Specific Phoenix version
mix archive.install hex phx_new #{specific-phoenix-version}
```

Reply:
Sure I will adjust that one.